PollinatorPathway/Controllers/UserProfileAPI.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using PollinatorPathway.Data;
+using PollinatorPathway.Model;
+
+// For more information on enabling Web API for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
+
+namespace PollinatorPathway.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class UserProfileAPI : ControllerBase
```

Comment:
Please rename this class to "Profiles" so that the access URL will better match the convention
`https://localhost:7221/api/profile`

Reply:
You can also add a  subfolder called "API" within the "Controllers" folder to place both of these API controllers